src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockToolbar/BlockToolbar.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import React from 'react';
-import classNames from 'classnames';
+import classnames from 'classnames';
```

Comment:
i think we imported this as `classNames` in all other componenets. not sure if there is a reason, but i think i would keep things consistent 🙈